content/en/docs/development/drupal-10-update/_index.md

```diff
@@ -0,0 +1,124 @@
+    - Some common issues:
+        - [Remove jQuery dependency from the once feature](https://www.drupal.org/node/3158256)
+        - [Migrating dependencies on core jQuery UI libraries](https://www.drupal.org/docs/upgrading-drupal/upgrading-from-drupal-8-or-later/upgrading-from-drupal-9-to-drupal-10-0/migrating-dependencies-on-core-jquery-ui-libraries)
+        - [Functions file_create_url() and file_url_transform_relative() are deprecated](https://www.drupal.org/node/2940031)
+        - [Access checking must be explicitly specified on content entity queries](https://www.drupal.org/node/3201242)
+        - [drupal_get_path() and drupal_get_filename() have been deprecated in favor of extension listing services](https://www.drupal.org/node/2940438)
+        - [Preparing for Twig 2 in Drupal 9](https://www.drupal.org/node/3071078)
+- Search for `hook_field_widget_form_alter`, `hook_field_widget_multivalue_form_alter`, `hook_field_widget_WIDGET_TYPE_form_alter` and `hook_field_widget_multivalue_WIDGET_TYPE_form_alter`. These hooks have been deprecated in Drupal 9.2 and not available anymore in Drupal 10. [Streamline field widget hooks](https://www.drupal.org/node/3180429)
+
+## Patches
+
+- Review patches in `composer.json`. Review any that are no longer applying or may be duplicated by the distribution.
+- Carefully review and re-roll custom patches.
+
+## Update
+
+At this point you should be ready to update to the latest version of the distribution:
+
+- Edit the `ycloudyusa` version in your project root `composer.json`: `"ycloudyusa/yusaopeny":"^10.3",`
+- Run `composer update`
+- If errors occur, review the conflicts, check out the [known issues](https://www.drupal.org/docs/updating-drupal/updating-drupal-core-via-composer#known-issues), and attempt to resolve them.
+- Re-run the previous steps until they complete successfully
+- Run `drush updb`, review the updates, and run them.
+
+## Smoke tests
+
+We recommend reviewing critical functionality after the update to ensure any custom functionality still works.
+
+## Troubleshooting
+
+### Composer issues
+
+Composer can be ... tricky. To resolve composer conflicts:
+
+- If specific modules conflict, try requiring them directly to get more information about the conflict.
+- Make good use of [composer why](https://getcomposer.org/doc/03-cli.md#depends-why) and [composer why-not](https://getcomposer.org/doc/03-cli.md#prohibits-why-not)
+
+### Update hook conflicts
+
+If you run into an error like this:
+
+> \>  [notice] Update started: y_lb_update_9011
+>
+> \> [error]  Configuration <em class="placeholder">core.entity_view_display.node.lb_event.featured</em> depends on the <em class="placeholder">core.entity_view_mode.node.featured</em> configuration that will not exist after import.
+
+you may be able to resolve it yourself:
+
+1. Search the codebase for the missing config.
```

Comment:
can you add an example of how to search?  i.e. `grep -rI lb_event .` from the `docroot` parent folder